packages/styles/src/components/chip/chip.stories.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  title: 'Patterns/Badges, Chips, and Tags',
+  argTypes: {
+    active: {
+      control: { type: 'boolean' },
+      description: 'Adds a modifier class indicating an active/selected chip.'
+    },
+  },
+};
+
+const ChipTemplate = (args) => {
+  //const active = (args.active ==true) ? 'true' : 'false';
+
+  return `
+    <button type="button" class="cbp-chip" ${args.active != undefined ? 'aria-pressed="'+args.active+'"' : ''}>
+      <span class="cbp-chip__label">Hotlist</span>
+      <div class="cbp-chip__icon">
+        <i class="fas fa-plus"></i>
+      </div>
+    </button>
+  `;
+};
```

Comment:
Similar to the badge story
- Destructure the `args` parameter
- Let's discuss this, does the `aria-pressed` attribute need to be hidden when the markup is first loaded? then after its clicked, it will set the attribute and set it to `true/false` on subsequent clicks. Can we just set the attribute when its first loaded? 
- Same as previous comment, if its only return the mark up and no variables inside the entire function we can replace the brackets with parentheses and delete the `return` statement.

Reply:
Actually there should be aria-pressed by default, so I went ahead and updated. I think I copied that from the segmented button group and that component sets aria-pressed automatically for the whole group.

Fixed the rest.